src/plugins/blocksmith/resolver.js

```diff
@@ -287,13 +287,13 @@ export const getPlaceholder = (type, props = {}) => {
       component = createLazyComponent(import('@triniti/cms/plugins/blocksmith/components/youtube-video-block-placeholder'));
       break;
     default:
-      return null;
+      component = createLazyComponent(import('@triniti/cms/plugins/blocksmith/components/default-block-placeholder'));
```

Comment:
@efjacobson - when does this default-block gets returned?

Reply:
@arunkarnati if something like what Joel fixed happens again (the copy paste with image). where a block is considered to be atomic by draft, but `message` is undefined here https://github.com/triniti/cms-js/blob/error-reporting-recovery/src/plugins/blocksmith/components/blocksmith/index.jsx#L608

Reply:
@efjacobson - IC, in the edge case you mentioned above, what will be the value of `blockData.get('canvasBlock').schema()`? `undefined`?

Reply:
@arunkarnati in this edge case there will be an `atomic` block that does not have a `canvasBlock` in its data. so yes, the `type` parameter for this function will be `undefined`.

Reply:
@arunkarnati in this edge case `blockData.get('canvasBlock').schema()` will throw

Reply:
@efjacobson - got it, thanks!